packages/tasks/src/local-apps.ts

```diff
@@ -113,6 +113,13 @@ export const LOCAL_APPS = {
 		displayOnModelPage: isGgufModel,
 		deeplink: (model) => new URL(`sanctum://open_from_hf?model=${model.id}`),
 	},
+	msty: {
+		prettyLabel: "Msty",
+		docsUrl: "https://msty.app",
+		mainTask: "text-generation",
+		displayOnModelPage: isGgufModel,
+		deeplink: (model) => new URL(`msty://models/search/hf/${model.id}`),
```

Comment:
btw take a look at https://github.com/huggingface/huggingface.js/pull/738 in case you'd like to support opening from an optional specific file inside the repo! thanks!

Reply:
Hi @julien-c, thank you for your comment! This would be a very useful feature indeed but would take some time to get it out. Is it possible to get this merged in the meantime and we can send another PR when we implement and test the specific file deep link?

Reply:
sounds good!